Mathlib/Algebra/Category/ModuleCat/Topology/Homology.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Richard Hill, Andrew Yang
+-/
+import Mathlib.Algebra.Category.ModuleCat.Abelian
+import Mathlib.Algebra.Category.ModuleCat.Topology.Basic
+import Mathlib.Algebra.Homology.ShortComplex.Abelian
+import Mathlib.Algebra.Homology.ShortComplex.PreservesHomology
+
+/-!
+
+# `TopModuleCat` is a `CategoryWithHomology`
+
```

Comment:
Could you add a comment saying that despite `TopModuleCat` not being `Abelian` in general, it has this property. (I did not expect we would have any instance of `CategoryWithHomology` outside of abelian categories!) (Also, would it makes sense to say that the key-ingredient in the proof is essentially that the two obvious topologies on a subquotient are the same? If so, it would be nice to add such a comment somewhere in this file.)